x-pack/plugins/fleet/common/services/datastream_es_name.ts

```diff
@@ -16,7 +16,7 @@ export function getRegistryDataStreamAssetBaseName(dataStream: {
   type: string;
   hidden?: boolean;
 }): string {
-  const baseName = `${dataStream.type}-${dataStream.dataset}`;
+  const baseName = `${dataStream.type}-${dataStream.dataset.toLowerCase()}`;
```

Comment:
Just a question: if I understand correctly, those templates from now on will be forced to have lowercase names. Could there be any risk in forcing this change?